### DISCUSSION TITLE
`agent-merging`: Move record to keep from the left hand to the right hand side of the screen

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5722] [Feature Requests]
To be reviewed.

**Requested By:** Ian Engelbrecht

**Comment by maxpatiiuk** [comment]
you mean move the first column to the right edge?
as a pref or a button or always?

**Comment by grantfitzsimmons** [comment]
Move to the right side instead of the left side

**Comment by CarolineDenis** [comment]
@grantfitzsimmons, what is the decision on this?

**Comment by grantfitzsimmons** [comment]
This would be nice as a preference, but not by default
